test/integration/addListing.js

```diff
@@ -77,34 +78,34 @@ module.exports = async (artifacts, web3, config, fcrJsConfig, utils) => {
   //   console.log(err, res);
   // });
 
-  console.log('----------------------- CREATING REGISTRY -----------------------')
-  const { registryProxy} = await utils.getProxiesBYO(token.address);
-  const registry = registryProxy;
+  // console.log('----------------------- CREATING REGISTRY -----------------------')
```

Comment:
Going to approve but do you want to keep all this commented out code?

Reply:
@cwhinfrey I have to fix all of this and add it back, I just want to get this merged quickly so that @emaG3m can run unit tests faster